### PR TITLE
Add zh to docs specific to zh

### LINKED ISSUE #25193: [zh] replace "(/docs)" to "(/zh/docs)"
**This is a Bug Report**
The doc link in zh content should be replaced from `/docs` to `/zh/docs`

**Proposed Solution:**

To find the file ,run the following  step:
```shell
cd website/content/zh
find ./ -type f -name "*.md" | xargs grep "(/docs"
```

### PR DESCRIPTION
**Pull request by championshuttler**
Signed-off-by: championshuttler <shivams2799@gmail.com>

Fixes #25193

### CODE REVIEW

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 1]
```diff
@@ -484,11 +484,11 @@ kubectl [flags]
...
-* [kubectl annotate](/docs/reference/generated/kubectl/kubectl-commands#annotate)	 - 更新资源所关联的注解
-* [kubectl api-resources](/docs/reference/generated/kubectl/kubectl-commands#api-resources)	 - 打印服务器上所支持的 API 资源
-* [kubectl api-versions](/docs/reference/generated/kubectl/kubectl-commands#api-versions)	 - 以“组/版本”的格式输出服务端所支持的 API 版本
-* [kubectl apply](/docs/reference/generated/kubectl/kubectl-commands#apply)	 - 基于文件名或标准输入，将新的配置应用到资源上
-* [kubectl attach](/docs/reference/generated/kubectl/kubectl-commands#attach)	 - 连接到一个正在运行的容器
+* [kubectl annotate](/zh/docs/reference/generated/kubectl/kubectl-commands#annotate)	 - 更新资源所关联的注解
+* [kubectl api-resources](zh/docs/reference/generated/kubectl/kubectl-commands#api-resources)	 - 打印服务器上所支持的 API 资源
+* [kubectl api-versions](/zh/docs/reference/generated/kubectl/kubectl-commands#api-versions)	 - 以“组/版本”的格式输出服务端所支持的 API 版本
+* [kubectl apply](/zh/docs/reference/generated/kubectl/kubectl-commands#apply)	 - 基于文件名或标准输入，将新的配置应用到资源上
+* [kubectl attach](/zh/docs/reference/generated/kubectl/kubectl-commands#attach)	 - 连接到一个正在运行的容器
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/kubectl.md:491]: these generated files don't have localized version.

File: content/zh/docs/test.md [hunk 2]
```diff
@@ -505,7 +505,7 @@ You can also use HTML, but it is not preferred.
   例如，原链接目标为 `/docs/foo/bar` 时，译文中的链接目标应为
   `/zh/docs/foo/bar`。例如：
 
-  - 英文版本链接 [Kubernetes Components](/docs/concepts/overview/components/)
+  - 英文版本链接 [Kubernetes Components](/zh/docs/concepts/overview/components/)
```
Comment (tengqm) [inline content/zh/docs/test.md:508]: This is intentionally English link.

File: content/zh/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume.md [hunk 3]
```diff
@@ -208,6 +208,6 @@ the shared Volume is lost.
 * 进一步了解[复合容器的模式](https://kubernetes.io/blog/2015/06/the-distributed-system-toolkit-patterns.html)
 * 学习[模块化架构中的复合容器](https://www.slideshare.net/Docker/slideshare-burns)
 * 参见[配置 Pod 使用卷来存储数据](/zh/docs/tasks/configure-pod-container/configure-volume-storage/)
-* 参考 [Volume](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#volume-v1-core)
-* 参考 [Pod](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#pod-v1-core)
+* 参考 [Volume](/zh/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#volume-v1-core)
+* 参考 [Pod](/zh/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#pod-v1-core)
```
Comment (tengqm) [inline content/zh/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume.md:212]: Things under /docs/reference/generated/... don't have Chinese localization.

**Comment by tengqm** [comment]
Thanks for the contribution. Please split this into smaller PRs (e.g. each PR focus on a section or subsection).
Huge PRs like this are difficult/impossible to review.

File: content/zh/blog/_posts/2019-08-06-OPA-Gatekeeper-Policy-and-Governance-for-Kubernetes.md [hunk 4]
```diff
@@ -79,7 +79,7 @@ Before we dive into the current state of Gatekeeper, let’s take a look at how
 --->
 * Gatekeeper v1.0 - 使用 OPA 作为带有 kube-mgmt sidecar 的许可控制器，用来强制执行基于 configmap 的策略。这种方法实现了验证和转换许可控制。贡献方：Styra
 * Gatekeeper v2.0 - 使用 Kubernetes 策略控制器作为许可控制器，OPA 和 kube-mgmt sidecar 实施基于 configmap 的策略。这种方法实现了验证和转换准入控制和审核功能。贡献方：Microsoft
- * Gatekeeper v3.0 - 准入控制器与 [OPA Constraint Framework](https://github.com/open-policy-agent/frameworks/tree/master/constraint) 集成在一起，用来实施基于 CRD 的策略，并可以可靠地共享已完成声明配置的策略。使用 kubebuilder 进行构建，实现了验证以及最终转换（待完成）为许可控制和审核功能。这样就可以为 [Rego](https://www.openpolicyagent.org/docs/latest/how-do-i-write-policies/) 策略创建策略模板，将策略创建为 CRD 并存储审核结果到策略 CRD 上。该项目是 Google，Microsoft，Red Hat 和 Styra 合作完成的。
+ * Gatekeeper v3.0 - 准入控制器与 [OPA Constraint Framework](https://github.com/open-policy-agent/frameworks/tree/master/constraint) 集成在一起，用来实施基于 CRD 的策略，并可以可靠地共享已完成声明配置的策略。使用 kubebuilder 进行构建，实现了验证以及最终转换（待完成）为许可控制和审核功能。这样就可以为 [Rego](https://www.openpolicyagent.org/zh/docs/latest/how-do-i-write-policies/) 策略创建策略模板，将策略创建为 CRD 并存储审核结果到策略 CRD 上。该项目是 Google，Microsoft，Red Hat 和 Styra 合作完成的。
```
Comment (zhaojizhuang) [inline content/zh/blog/_posts/2019-08-06-OPA-Gatekeeper-Policy-and-Governance-for-Kubernetes.md:82]: this link `https://www.openpolicyagent.org/zh/docs/latest/how-do-i-write-policies/` is english site, don't replace

File: content/zh/blog/_posts/2015-04-00-Weekly-Kubernetes-Community-Hangout_17.md [hunk 5]
```diff
@@ -192,7 +192,7 @@ Notes
 
 
 [1]: http://kubernetes.io/images/nav_logo.svg
-[2]: http://kubernetes.io/docs/
+[2]: http://kubernetes.io/zh/docs/
 [3]: https://kubernetes.io/blog/
```
Comment (helight) [inline content/zh/blog/_posts/2015-04-00-Weekly-Kubernetes-Community-Hangout_17.md:196]: 这个地址也有中文地址呀

**Comment by tengqm** [comment]
@championshuttler Please help resolve the conflicts. It would also help if you can split this into smaller PRs for ease of review. Thanks.

**Comment by tengqm** [comment]
No response from the author for one month; the PR has involved too many changes.
/close

**Pull request closed without merging**